eng/pipelines/docindex.yml

```diff
@@ -84,7 +106,7 @@ jobs:
         parameters:
           BaseRepoBranch: $(DefaultBranch)
           BaseRepoOwner: $(DocRepoOwner)
-          CommitMsg: "Update docs CI configuration"
+          CommitMsg: "Update docs CI configuration Build: $(System.CollectionUri)$(System.TeamProject)/_build/results?buildId=$(Build.BuildId)"
```

Comment:
Do we want this in the commit history? As a lot of these links will die overtime as builds expire. Perhaps only in the PR message?

Reply:
There isn't a PR message in this step. This step pushes directly to `main` and while the logs for the docindex runs will go away over time, it's a useful to tie onboarding information directly to docindex pipeline runs when investigating.